src/org/rascalmpl/test/infrastructure/RascalJUnitTestRunner.java

```diff
@@ -213,14 +213,14 @@ public Description getDescription() {
 
                     // the order of the tests aren't decided by this list so no need to randomly order them.
                     for (AbstractFunction f : tests) {
-                        modDesc.addChild(Description.createTestDescription(clazz, computeTestName(f.getName(), f.getAst().getLocation())));
+                        modDesc.addChild(Description.createTestDescription(module, computeTestName(f.getName(), f.getAst().getLocation())));
```

Comment:
is this related to this PR?

Reply:
No this was an accident

Reply:
No this was an accident

> The RascalJUnitTestRunner changes confuse me a bit.

well, I was confused when this happened, so that makes total sense again. I accidentally merged this change into the PR. I've tested it and there is no effect, positive or negative, to the test execution other than that the right module names are now shown when a test fails in the reports (also in the XML files).

Reply:
that's nice :) okay, well, I'll allow it ;) ;)